app/controllers/request-to-go-live/organisation-name/post.controller.js

```diff
@@ -6,12 +6,15 @@ const lodash = require('lodash')
 // Local dependencies
 const goLiveStageToNextPagePath = require('../go-live-stage-to-next-page-path')
 const { requestToGoLive } = require('../../../paths')
+const { validateOrganisationName } = require('../../../utils/organisation_name_validation')
```

Comment:
Great that you moved the logic to a separate file 👍 I think we use dashes for file delimiters when we were discussing last time

Reply:
I'm not sure why but we seem to use snake-casing in the utils folder

Reply:
Ah, ok, I didn't notice that, let's keep it consistent per directory then